scripts/remove-uncommited-changes.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+set -e
+GIT_STATUS="$(git status --procelain)"
+VERSION="$(./release/relman.py --check-current --show-head-version)"
+if [ "$GIT_STATUS" != "" ]
+then
+    git checkout -f -- "./stable/insights/Chart.lock"
```

Comment:
You just want to remove any uncommitted changes to this particular file. Why don't you just execute this unconditionally? The end result is the same, which is that there are no uncommitted changes to this file. You can probably just remove this script and execute this command directly in the workflow steps.